src/java.xml/share/classes/javax/xml/catalog/CatalogFeatures.java

```diff
@@ -175,21 +177,17 @@
  * is not necessary or will be ignored.
  * <p>
  * Properties set through the Catalog API override those that may have been set
- * by system properties and/or in {@code jaxp.properties}. In case of multiple
- * interfaces, the latest in a procedure shall take preference. For
- * {@link Feature#FILES}, this means that the URI(s) specified through the methods
+ * by system properties and/or in the
+ * <a href="../../../module-summary.html#ConfigurationFile">configuration file</a>.
+ * In case of multiple interfaces, the latest in a procedure shall take preference.
+ * For {@link Feature#FILES}, this means that the URI(s) specified through the methods
```

Comment:
"multiple interfaces" and "the latest in a procedure" aren't very clear here. If the setup for this section documents the precedence order more clearly then it will make it easier to word this section.

Reply:
We have two options here:
    - drop the discussion about "multiple interfaces", and leave it to the CatalogManager to make it clear;
    - add some sample code 
    
While we'll be adding a general Scope and Order, this is specific to the Catalog API. I'll try option 1.